### PR TITLE
[EB-1388] Validate for 'undefined' token on resetPasswordAsync

### PR DESCRIPTION
**Pull request by vdanylov**

### CODE REVIEW

File: yarn-error.log [hunk 1]
```diff
@@ -102,7 +102,11 @@ Trace:
   Error: Command failed.
   Exit code: 1
   Command: sh
+<<<<<<< HEAD
   Arguments: -c jest --testPathIgnorePatterns=__integration__ src/data/notification/__test__/notification.test.js
+=======
+  Arguments: -c jest --testPathIgnorePatterns=__integration__ -t 'resetPasswordAsync throw error'
+>>>>>>> [EB-1388] Validate for 'undefined' token on resetPasswordRequestAsync
```
Comment (maartenhemmes) [inline yarn-error.log]: @vdanylov are these conflicts, if so, please resolve them.

File: src/data/account/__test__/account.test.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ import {
   validateResetPasswordTokenAsync,
```
Comment (maartenhemmes) [inline src/data/account/__test__/account.test.js]: Other unit tests are failing now: @vdanylov 

  ● Test for reset password token validation › reject with statusCode 404 if the reset password is invalid

    Error
      [object Object] thrown

Comment (vdanylov) [reply src/data/account/__test__/account.test.js]: @maartenhemmes strange I fire this test and it passes.

Comment (vdanylov) [reply src/data/account/__test__/account.test.js]: @maartenhemmes I think You have got old version, cause I resolved these conflicts before. I'll check it one more time.

Comment (maartenhemmes) [reply src/data/account/__test__/account.test.js]: ok. on dev it's working fine, in this PR it's failing:

screen shot 2018-04-21 at 7 24 18 am

Comment (maartenhemmes) [reply src/data/account/__test__/account.test.js]: screen shot 2018-04-21 at 7 23 31 am

Comment (vdanylov) [reply src/data/account/__test__/account.test.js]: @maartenhemmes thanks, I debugged problem and fix it.

**Pull request merged**